code/modules/shuttle/spaceship_navigation_beacon.dm

```diff
@@ -0,0 +1,63 @@
+/obj/item/circuitboard/machine/spaceship_navigation_beacon
+	name = "Bluespace Navigation Gigabeacon (Machine Board)"
+	build_path = /obj/machinery/spaceship_navigation_beacon
+	req_components = list()
```

Comment:
Is it built without components intentionally?

Reply:
yes.